docs/qa/CometBFT-QA-38.md

```diff
@@ -0,0 +1,187 @@
+transactions.
+
+![mempool-avg](img38/200nodes/avg_mempool_size.png)
+
+The peaks observed coincide with the moments when some nodes reached round 1 of
+consensus (see below).
+
+The behavior is similar to the observed in the baseline, presented next.
+
+![mempool-cumulative-baseline](img37/200nodes_cmt037/mempool_size.png)
+
+![mempool-avg-baseline](img37/200nodes_cmt037/avg_mempool_size.png)
+
+
+### Peers
+
+The number of peers was stable at all nodes. It was higher for the seed nodes
+(around 140) than for the rest (between 20 and 70 for most nodes). The red
+dashed line denotes the average value.
+
+![peers](img38/200nodes/peers.png)
+
+Just as in the baseline, shown next, the fact that non-seed nodes reach more
+than 50 peers is due to [\#9548].
+
+![peers](img37/200nodes_cmt037/peers.png)
+
+
+### Consensus Rounds per Height
+
+Most heights took just one round, that is, round 0, but some nodes needed to
+advance to round 1.
+
+![rounds](img38/200nodes/rounds.png)
+
+The following specific run of the baseline required some nodes to reach round 1.
+
+![rounds](img37/200nodes_cmt037/rounds.png)
+
+
+### Blocks Produced per Minute, Transactions Processed per Minute
+
+The following plot shows the rate in which blocks were created, from the point
+of view of each node. That is, it shows when each node learned that a new block
+had been agreed upon.
+
+![heights](img38/200nodes/block_rate.png)
+
+For most of the time when load was being applied to the system, most of the
+nodes stayed around 20 blocks/minute.
+
+The spike to more than 100 blocks/minute is due to a slow node catching up.
+
+The baseline experienced a similar behavior.
+
+![heights-baseline](img37/200nodes_cmt037/block_rate.png)
+
+The collective spike on the right of the graph marks the end of the load
+injection, when blocks become smaller (empty) and impose less strain on the
+network. This behavior is reflected in the following graph, which shows the
+number of transactions processed per minute.
+
+![total-txs](img38/200nodes/total_txs_rate.png)
+
+The following is the transaction processing rate of the baseline, which is
+similar to above.
+
+![total-txs-baseline](img37/200nodes_cmt037/total_txs_rate.png)
+
+
+### Memory Resident Set Size
+
+The following graph shows the Resident Set Size of all monitored processes, with
+maximum memory usage of 1.6GB, slighlty lower than the baseline shown after.
+
+![rss](img38/200nodes/memory.png)
+
+A similar behavior was shown in the baseline, with even a slightly higher memory
+usage.
+
+![rss](img37/200nodes_cmt037/memory.png)
+
+The memory of all processes went down as the load is removed, showing no signs
+of unconstrained growth.
+
+
+### CPU utilization
+
+The best metric from Prometheus to gauge CPU utilization in a Unix machine is
+`load1`, as it usually appears in the [output of
+`top`](https://www.digitalocean.com/community/tutorials/load-average-in-linux).
+
```

Comment:
Here, we need to explain that this metric shows that the 3x signature check has no visible impact on the CPU usage, meaning that signature verification is not the bottleneck in a large-scale network.

I'll prepare a small commit and push it to this branch, if you agree